apple/OmnivoreKit/Sources/Models/LinkedItemSort.swift

```diff
@@ -30,23 +27,21 @@ public extension LinkedItemSort {
       return "sort:saved"
     case .oldest:
       return "sort:saved-ASC"
-//    case .recentlyRead:
-//      return "sort:updated"
+    case .recentlyRead:
+      return "sort:read"
     case .recentlyPublished:
       return "sort:published"
-//    case .relevance:
-//      return "relevance"
     }
   }
 
   var sortDescriptors: [NSSortDescriptor] {
     switch self {
-    case .newest /* , .relevance */:
+    case .newest:
       return [NSSortDescriptor(keyPath: \LinkedItem.savedAt, ascending: false)]
     case .oldest:
       return [NSSortDescriptor(keyPath: \LinkedItem.savedAt, ascending: true)]
-//    case .recentlyRead:
-//      return [NSSortDescriptor(keyPath: \LinkedItem.updatedAt, ascending: false)]
+    case .recentlyRead:
+      return [NSSortDescriptor(keyPath: \LinkedItem.updatedAt, ascending: false)]
```

Comment:
I think you want `.readAt` here

Reply:
oh nice. didn't know we had that now 👍🏽

Reply:
@satindar oh, i think you have to add it 😬

Reply:
Sorry, thought you were fetching that and storing in LinkedItem already.